Formula/zork.rb

```diff
@@ -0,0 +1,21 @@
+class Zork < Formula
+  desc "The Dugeon modified from FORTRAN to C"
+  homepage "https://github.com/devshane/zork"
+  url "https://github.com/devshane/zork/archive/v1.0.0.tar.gz"
+  version "1.0.0"
+  sha256 "ceedb436cf552bd8ca39abd3e85bae986c3f5df6f31c1bf2806d1b4c8509da67"
+  head "https://github.com/devshane/zork.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "make", "LIBDIR=#{lib}", "BINDIR=#{bin}"
+    bin.install "zork"
+    lib.install "dtextc.dat"
+    man6.install "dungeon.6"
```

Comment:
You can override these locations like you did above:
```ruby
system "make", "install", "LIBDIR=#{lib}", "BINDIR=#{bin}"
```

There’s still an issue because the Makefile doesn’t ensure those locations exist before writing in them. You thus have to create them before:
```ruby
bin.mkpath
lib.mkpath
system "make", "install", "LIBDIR=#{lib}", "BINDIR=#{bin}"
```